code/modules/mob/living/carbon/human/say.dm

```diff
@@ -106,7 +111,7 @@
 	var/list/returns[3]
 	var/speech_problem_flag = 0
 
-	if(silent || (sdisabilities & MUTE))
+	if(silent || (sdisabilities & MUTE) || !IsVocal())

```

Comment:
Yeah, no.
This goes in it's own conditional, in https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/mob/living/say.dm#70